### PR TITLE
Add support for Piwik

### PR DESCRIPTION
**Pull request by bnjbvr**
Hi! Thank you for this awesome pelican theme. Please find here a PR that allows one to support Piwik integration (it's like Google Analytics but self-hosted), if the PIWIK_SITE_ID and PIWIK_URL are set in the configuration.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -98,5 +98,25 @@
   <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid={{ ADD_THIS_ID }}" async="async"></script>
   {% endif %}
   {% block additional_js %}{% endblock %}
+
+  {% if PIWIK_URL %}
```
Comment (alexandrevicenzi) [inline templates/base.html]: Why not an `and`? I think it's better the 2 ifs.

**Comment by alexandrevicenzi** [comment]
Hi @bnjbvr, thanks for this integration.

Could you just take a look in my comment?

**Comment by bnjbvr** [comment]
@alexandrevicenzi indeed, I didn't know I could do that. I've updated my commit, can you take another look, please? Cheers!

**Comment by alexandrevicenzi** [comment]
Thanks @bnjbvr.

Could you update the [wiki](https://github.com/alexandrevicenzi/Flex/wiki)? There's a place for config variables :smile:

**Comment by bnjbvr** [comment]
And that's done. Thank you for merging.

**Comment by alexandrevicenzi** [comment]
Nice @bnjbvr :wink:

**Pull request merged**